### PR TITLE
Add a manually written pot file for the open status formatter

### PR DESCRIPTION
**Pull request by oshi97**
We want to be able to get translations from smartling instead of
asking a hitchhiker/native speaker at yext. Eventually we will
probably want some way of injecting translations into the formatter.

Also fix rename open-status-18n to open-status-i18n (year old typo).

J=SLAP-1455
TEST=none

### CODE REVIEW

File: translations/open-status.pot [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+msgctxt "Whether a store is open or closed"
+msgid "Open 24 Hours"
+msgstr ""
+
+msgctxt "Whether a store is open or closed"
+msgid "Closed"
+msgstr ""
+
+msgctxt "Whether a store is open or closed"
```
Comment (cea2aj) [inline translations/open-status.pot:13]: nit: This context might be a little confusing for this particular translation. How about something like: "Indicates the time that a store will open"

File: translations/open-status.pot [hunk 2]
```diff
@@ -0,0 +1,23 @@
+msgid ""
+msgstr ""
+"Content-Type: text/plain; charset=UTF-8\n"
+
+msgctxt "Whether a store is open or closed"
```
Comment (cea2aj) [inline translations/open-status.pot:5]: what do you think about something like: "The hours that a store is open"

Comment (oshi97) [reply translations/open-status.pot]: updated!

File: translations/open-status.pot (hunk 1 above)
Comment (oshi97) [reply translations/open-status.pot]: yeah makes sense

**Review by cea2aj — APPROVED** [review]

**Pull request merged**